examples/demos/components/Editors/DropDownEditor.tsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef, useCallback, useImperativeHandle, useState } from 'react';
+import { Column, Editor } from '../../../../src';
+
+interface Option {
+  id: string;
+  title: string;
+  value: string;
+  text: string;
+}
+
+interface DropDownEditorProps<TRow> {
+  column: Column<TRow>;
+  options: Array<Option | string>;
+  value?: string;
+  onBlur(): void;
+}
+
+type DropDownEditorHandle = Pick<Editor<{[key: string]: string | undefined}>, 'getInputNode' | 'getValue'>;
+
+function DropDownEditorWithRef<TRow>({ column, value, onBlur, options }: DropDownEditorProps<TRow>, ref: React.Ref<DropDownEditorHandle>): JSX.Element {
+  const selectRef = React.createRef<HTMLSelectElement>();
```

Comment:
You're creating a new ref on each render.
```suggestion
  const selectRef = useRef<HTMLSelectElement>(null);
```

Reply:
shoot, forgot change this after converting from a class component